Src/CSharpier.Tests/FormattingTests/SetUpFixture.cs

```diff
@@ -10,16 +10,26 @@ public class SetUpFixture
     [OneTimeSetUp]
     public void OneTimeSetUp()
     {
-        EmptyFiles.Extensions.AddTextExtension(".test");
+        EmptyFiles.FileExtensions.AddTextExtension(".test");
+
         DiffTools.AddToolBasedOn(
             DiffTool.WinMerge,
             name: "WritableWinMerge",
-            arguments: (temp, target) =>
-            {
-                var leftTitle = Path.GetFileName(temp);
-                var rightTitle = Path.GetFileName(target);
-                return $"/u /wr /e \"{temp}\" \"{target}\" /dl \"{leftTitle}\" /dr \"{rightTitle}\"";
-            }
+            launchArguments: new(LeftArguments, RightArguments)
         );
     }
+
+    static string LeftArguments(string temp, string target)
+    {
+        var tempTitle = Path.GetFileName(temp);
+        var targetTitle = Path.GetFileName(target);
+        return $"/u /wr /e \"{target}\" \"{temp}\" /dl \"{targetTitle}\" /dr \"{tempTitle}\" /cfg Backup/EnableFile=0";
+    }
+
+    static string RightArguments(string temp, string target)
+    {
+        var tempTitle = Path.GetFileName(temp);
+        var targetTitle = Path.GetFileName(target);
+        return $"/u /wl /e \"{temp}\" \"{target}\" /dl \"{tempTitle}\" /dr \"{targetTitle}\" /cfg Backup/EnableFile=0";
+    }
```

Comment:
The files show up in winmerge properly, but these changes result in the `TestCase.actual.test` file being editable, when it should be `TestCase.test` that is editable.

Or in other words "Copy from Left" is an option, but should not be, and "Copy to Left" is unavailable but should be.

![image](https://github.com/user-attachments/assets/7763b4df-7ad9-4355-ba6f-98b278ae0e60)

Reply:
Ah... Probably temp and target need to change the position..